db/schema.rb

```diff
@@ -100,9 +100,10 @@
   create_table "application_scope_limitations", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.uuid "legal_aid_application_id"
     t.uuid "scope_limitation_id"
-    t.boolean "substantive", default: true
```

Comment:
I'm not sure these changes to `schema.rb` relate to your other changes in this PR?